internal/cmd/controller/grutil/resourcekey.go

```diff
@@ -27,37 +28,39 @@ func bundleErrorState(summary fleet.BundleSummary) string {
 	return bundleErrorState
 }
 
-// fromResourceKey lists all bundles for this GitRepo and returns a list of
+// fromResourceKey lists all bundlesDeployments for this GitRepo and returns a list of
```

Comment:
Sorry, just spotted this typo :see_no_evil: 
```suggestion
// fromResourceKey lists all bundledeployments for this GitRepo and returns a list of
```
or
```suggestion
// fromResourceKey lists all bundle deployments for this GitRepo and returns a list of
```

Reply:
Ugh, thanks for the correction :laughing: